scripts/local.sh

```diff
@@ -35,7 +35,7 @@ startFrontend() {
 stopBackend() {
   if lsof -i:8080 -t &> /dev/null; then
     printf "\n======== Stopping backend ========\n"
-    kill "$(lsof -i:8080 -t)"
+    pkill java
```

Comment:
@LittleBaiBai this is very aggressive - it'll kill any java process on the machine. Probably not what we want?

Reply:
You are right! I updated that.